index.bs

```diff
@@ -3351,7 +3351,7 @@ Candidate Amendments</h4>
 	to be updated quickly with the Working Group's most current thinking,
 	even when the [=candidate amendments=] have not yet received
 	sufficient review or implementation experience
-	to be normatively incorporated into the specification proper.
+	to be normatively incorporated into the specification properly.
```

Comment:
`proper`, here, is not an adverb modifying `incorporated`. It is an adjective modifying `the specification`, meaning the central/core/essential part of it.
```suggestion
	to be normatively incorporated into the specification proper.
```